### ISSUE TITLE
Rewrite the future work and limitations sections

### ISSUE THREAD

**Issue by j---** [CERTCC/SSVC#38]
This is a one issue because future work and limitations sections are closely related. Future work is problems we can fix. Limitations is problems we know we can't fix because we have made a choice to prioritize something else. 

Both of these sections will need to be updated to make sure that all items identified as future work in the rest of the document are collated. And in general, the v2 updates will change what needs to be done in the future and may adjust the limitations. In that sense, a final pass on these sections needs to be one of the last things done in v2.

- [x] Collate items identified as future work in the document to make sure nothing flagged as future work in prior sections is left out of the future work section. The main section headings in the Future Work section should still be used to emphasize the most important items. 
- [x] After most v2 work is done, edit / rewrite the future work section
- [x] After most v2 work is done, edit / rewrite the limitations section

**Comment by laurie-tyz** [comment]
List of "Future Work" 

- [ ] VRDA is likely adequate for national-level CSIRTs that do general CVD, but other CSIRT types may have different needs. Future work may elicit those types and make a few different decision options.

- [ ] Specialized coordination organizations exist (e.g., ICS-CERT, which conducts CVD for safety-critical systems). We have not developed a coordination tree in this work, but future work could use our principles and design techniques to refine and evaluate VRDA or some other decision tree for coordinated vulnerability disclosure.

- [ ] Different risk tolerance or risk discounting postures are not addressed in the current work; accommodating such tolerance or discounting explicitly is an area for future work.

- [ ] However, future work should look for and prevent large mismatches between the outputs of the utility decision point and the exploit markets.

- [ ] Aggregation suggests that the stakeholder’s response to this decision point cannot be less than the most severe credible safety impact, but we leave the specific aggregation method or function as a domain-specific extension for future work.

- [ ] The factors that influence the safety impact level are diverse. This paper does not exhaustively discuss how a stakeholder should answer a question; that is a topic for future work.

- [ ] The factors that influence the mission impact level are diverse. This paper does not exhaustively discuss how a stakeholder should answer a question; that is a topic for future work.

- [ ] This paper focuses only on the structure of decision making. Advice about how to search for information about a vulnerability is a separate project that will be part of future work. In some domains, namely exploit availability, we have started that work in parallel.

- [ ] Future work on SSVC as well as further development of other prioritization methods can benefit from using the method described in the pilot. Future instances should use more representative participant demographics.

- [ ] We are not presenting this section as the final word on excluding these ideas, but we hope the reasons for excluding them are instructive, will help prevent others from re-inventing the proverbial wheel, and can guide thinking on future work.

**Comment by ahouseholder** [comment]
From a comment on #119:

- [ ] Internationalization/localization of the vector and JSON formats

The actual future work to be done starts with #123

**Comment by j---** [comment]
Also solve #66 during this.

**Comment by j---** [comment]
> We have not developed a coordination tree in this work, 

v2 does in fact do this. So I'm marking this as "done" even though it's not going in the future work section (because it was added by #11 ). 

> how a stakeholder should answer a question

Similarly, I'm calling these two completed by #27 

> Advice about how to search for information

And this was included in the document under #33 

> Future instances should use more representative participant demographics.

This is stated in the current future work section, and remains true, but nothing to add.
